### PR TITLE
add inference test for mosaicml-mpt-7b-chat

### PR DESCRIPTION
**Pull request by jiafuzha**
## Type of Change

CI for mpt-7b-chat

## Description

CI test based on docker

## Expected Behavior & Potential Risk

CI tests run successfully on all supported models so far

## How has this PR been tested?

CI tests triggered and run in self hosted runner

## Dependency Change?

no

### CODE REVIEW

**Comment by jiafuzha** [comment]
@XuehaoSun @chensuyue please help review.

It doesn't work when try to read env variables from runner .env. So I added some logic to load env variables from the .env.

File: .github/workflows/chatbot-test.yml [hunk 1]
```diff
@@ -25,3 +26,7 @@ jobs:
   call-inference-llama-2-7b-chat-hf:
     uses: ./.github/workflows/chatbot-inference-llama-2-7b-chat-hf.yml
 
+  call-inference-mosaicml-7b-chat:
```
Comment (chensuyue) [inline .github/workflows/chatbot-test.yml:29]: call-inference-mpt-7b-chat

File: .github/workflows/chatbot-test.yml [hunk 2]
```diff
@@ -25,3 +26,7 @@ jobs:
...
+    uses: ./.github/workflows/chatbot-inference-mosaicml-mpt-7b-chat.yml
```
Comment (chensuyue) [inline .github/workflows/chatbot-test.yml:30]: chatbot-inference-mpt-7b-chat.yml

File: .github/workflows/chatbot-test.yml (hunk 1 above)
Comment (jiafuzha) [reply .github/workflows/chatbot-test.yml:29]: good catch

File: .github/workflows/chatbot-test.yml (hunk 2 above)
Comment (jiafuzha) [reply .github/workflows/chatbot-test.yml:30]: good catch

File: .github/workflows/chatbot-inference-llama-2-7b-chat-hf.yml [hunk 3]
```diff
@@ -16,18 +15,21 @@ jobs:
       - name: Checkout
         uses: actions/checkout@v2
 
+      - name: Load environment variables
+        run: cat ~/itrex-actions-runner/.env >> $GITHUB_ENV
```
Comment (chensuyue) [inline .github/workflows/chatbot-inference-llama-2-7b-chat-hf.yml:19]: Need to find a way to hide the token in logs.

File: .github/workflows/chatbot-test.yml (hunk 1 above)
Comment (jiafuzha) [reply .github/workflows/chatbot-test.yml]: renamed

File: .github/workflows/chatbot-test.yml (hunk 2 above)
Comment (jiafuzha) [reply .github/workflows/chatbot-test.yml]: renamed

File: .github/workflows/chatbot-inference-llama-2-7b-chat-hf.yml (hunk 3 above)
Comment (jiafuzha) [reply .github/workflows/chatbot-inference-llama-2-7b-chat-hf.yml:19]: I'll try to fix it in coming PRs.

**Review by chensuyue — APPROVED** [review]

**Review by XuehaoSun — APPROVED** [review]

**Review by lvliang-intel — APPROVED** [review]

**Pull request merged**